src/codegen/sys/ffi_type.rs

```diff
@@ -92,7 +92,7 @@ fn ffi_inner(env: &Env, tid: library::TypeId, mut inner: String) -> Result {
                 Type => "GType",
                 Pointer => {
                     match &inner[..] {
-                        "void" => "c_void",
+                        "void" => "gpointer",
```

Comment:
If you look 3 lines above, this is already the `Pointer` case and for resolving the "inner" part of the pointer. The inner part is `c_void`, but the outer part (`*mut` or `*const*) is for some reason not made use of. Or I need to read this code in more detail later :)